Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN if [ "$SKIP_TESTS" = 'false' ]; then \
   npm install cspell && npm run spellcheck; fi
 
 #-----------------------------
-FROM otomi/tools:v1.4.16 as test
+FROM otomi/tools:binzx as test
```

Comment:
Needs to be replaced with new version of otomi tools

Reply:
Please make PR that updates tools then we can use tools image from master

Reply:
https://github.com/redkubes/otomi-core/pull/452

Reply:
What about just one Dockerfile? I am frustrated by building and maintaining them separately. 

Also the permissions between these images are weird and not preserved as defined in the tools image, see:

```
root@ubuntu-16gb-nbg1-1:/home/sv/otomi/otomi-core# ls -lah /home/app
total 52K
drwxr-xr-x 1 app  app  4.0K Jul  2 07:56 .
drwxr-xr-x 1 root root 4.0K Jul  2 07:56 ..
drwxr-xr-x 2 1000 1000 4.0K Jun 17 15:05 .aws
drwxr-xr-x 2 1000 1000 4.0K Apr 13 13:32 .azure
drwxr-xr-x 1 app  app  4.0K Jun 22 11:55 .cache
drwxr-xr-x 4 root root 4.0K Jul  2 07:56 .config
drwxr-xr-x 2 root root 4.0K Jul  2 07:56 .kube
drwxr-xr-x 1 app  app  4.0K Jun 22 11:55 .local
drwxr-xr-x 1 app  app  4.0K Jul  1 14:36 .npm
drwxrwxr-x 2 1000 1000 4.0K May 21 06:01 .ssh
drwxr-xr-x 1 app  app  4.0K Jul  1 14:36 stack
drwxr-xr-x 1 app  app  4.0K Jun 22 11:55 tools
```

Reply:
Discussion for somewhere else, closing